queue_entry.py

```diff
@@ -553,7 +552,7 @@ def execute(self):
             # Create a centred postions of the current postion
             pos_dict = self.diffractometer_hwobj.getPositions()
             cpos = queue_model_objects.CentredPosition(pos_dict)
-            pos = shape_history.Point(None, cpos, None) #, True)
+            #pos = shape_history.Point(None, cpos, None) #, True)
```

Comment:
Sorry to come back on this...
I would be happy to merge the PR and to test it here (as we are in our machine day without users),
but it won't work: look at line 563 it does `pos.qub_point` etc.
Maybe we can just remove the lines 563 to 566 ?
Any hint would be appreciated.